java-runtimes-common/test-spring-application/pom.xml

```diff
@@ -25,6 +25,8 @@
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <deployment.token>default_deployment_token</deployment.token>
     <packaging.type>jar</packaging.type>
+    <app.deploy.runtime>java</app.deploy.runtime>
+    <app.deploy.image></app.deploy.image>
```

Comment:
why do we need an empty property to be defined?

Reply:
👍  Can be omitted. FYI if the property is missing, the template variable will remain in the generated Dockerfile, but come to think of it, this is actually better than what I have here, where it will just be filled in with an empty string.